package.json

```diff
@@ -17,7 +17,9 @@
     "@types/node": "^8.5.9",
     "coveralls": "^3.0.0",
     "cz-conventional-changelog": "^2.1.0",
-    "lerna": "^2.8.0"
+    "lerna": "^2.8.0",
+    "@types/uuid": "^3.4.3",
+    "debug": "^3.1.0"
```

Comment:
I find these two new entries rather suspicious. First of all, they are not sorted alphabetically. Secondly, the top-level `package.json` files should contain only dev-dependencies related to overal project setup. Everything else should go to individual package package.json, e.g. `packages/repository/package.json`.

I'll fix this problem myself to speed up landing of this patch.